### DISCUSSION TITLE
Multiple widgets

### DISCUSSION THREAD

**Discussion by jonmountjoy** [ABausG/home_widget#307] [Show and tell]
Hello,

Thank you for this lovely package.  I wonder widgets and intents are becoming more important in the mobile world, and so this is important.

Do you have any advice/documentation/recommendations about how to add multiple widgets (different widgets, that do different things)?

Thanks

**Comment by jonmountjoy** [comment]
The answer appears to be: yes, you can create multiple widgets.  Just create more, like you do the first one. 🕺

**Reply by sreerag2460** [reply]
Can you please explain or specify the things?
